R/system.R

```diff
@@ -1029,6 +1029,61 @@ getDBConnection <- function(type, host = NULL, port = "", databaseName = "", use
                              bigint = "numeric")
       connection_pool[[key]] <- conn
     }
+  } else if (type == "snowflake") {
+    # If the platform is Linux, set the below predefined driver installed on Collaboration Server
+    # so that this data source can be scheduled.
```

Comment:
Add comment on how this works - it overwrites the value for desktop when run on server.

Reply:
Added comment.